shell/platform/glfw/text_input_plugin.cc

```diff
@@ -134,7 +148,25 @@ void TextInputPlugin::HandleMethodCall(
                     "Could not set client, missing arguments.");
     }
     int client_id = client_id_json.GetInt();
```

Comment:
I wasn't sure what the convention was here - I initially used a local in case the following code aborts for some reason. I'll remove the local though.